GVFS/GVFS.Service/Handlers/RequestHandler.cs

```diff
@@ -0,0 +1,96 @@
+﻿using GVFS.Common.NamedPipes;
+using GVFS.Common.Tracing;
+using System.Runtime.Serialization;
+
+namespace GVFS.Service.Handlers
+{
+    public class RequestHandler
+    {
+        protected string requestDescription;
+
+        private const string MountRequest = "mount";
+        private const string UnmountRequest = "unmount";
+        private const string RepoListRequest = "repo list";
+        private const string UnknownRequest = "unknown";

```

Comment:
I would add `Name` or `Description` to the end of each of these `...Request` fields.